### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Sem-the-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
-# documentation
\ No newline at end of file
+# Our Neighbourhood API
+
+### About
+
+Our Neighbourhood API allows you to store and look up data on your local neighbourhood including people's address, number of people in each household, their names and age.
+
+Our Neighbourhood API is REST-based, mainly using POST and GET.
```
Comment (ros1995) [inline README.md:7]: Good mention that it is REST based

File: README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+    owner_last_name VARCHAR(45) NOT NULL,
+    owner_age INT NOT NULL,
+    street_address VARCHAR(40) NOT NULL,
+    postcode VARCHAR (10) NOT NULL,
+    number_people_in_household INT
+    
+);
+
+
+### see table
```
Comment (ros1995) [inline README.md:89]: Great summary table!

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+
+
+
+Method: GET
+Summary: people search
+Description: find people within certain age brackets and with specific household sizes
+
+
+
+### Field entries (schema)
```
Comment (ros1995) [inline README.md:66]: Maybe a short sentence to explain why you thought SQL database was the best option

File: README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+"errors": [
+{
+"status": 404,
+"title": "Route Not Found"
+}
+]
+}```
+
+
+### Adding information
```
Comment (roselynle) [inline README.md:44]: You should list the routes you will need with their HTTP verb and path

File: README.md [hunk 5]
```diff
@@ -1 +1,91 @@
...
+
+Method: POST
```
Comment (ros1995) [inline README.md:46]: Great start on the type of requests that the API will be handling!
Could possibly add different query params if you have more time

File: README.md [hunk 6]
```diff
@@ -1 +1,91 @@
...
+
+Our Neighbourhood API returns reults in various formats including JSON.
+
+### Base 
+https://our-neighbourhood-api/
+
+
+
+
+### Authentication
```
Comment (ros1995) [inline README.md:17]: Great that you included this authentication section, it's very clear to understand

File: README.md [hunk 7]
```diff
@@ -1 +1,91 @@
...
+### About
+
+Our Neighbourhood API allows you to store and look up data on your local neighbourhood including people's address, number of people in each household, their names and age.
+
+Our Neighbourhood API is REST-based, mainly using POST and GET.
+
+Our Neighbourhood API returns reults in various formats including JSON.
+
+### Base 
+https://our-neighbourhood-api/
```
Comment (ros1995) [inline README.md:12]: Good to see this is included because it's important and easily forgettable (like we did!)

File: README.md [hunk 8]
```diff
@@ -1 +1,91 @@
...
+
+
+### Errors
+
+When an error occurs during an API request, you will receive:
+
+An HTTP error status (in the 400-500 range)
+A JSON response containing more information about the error
+
+A typical error response looks like this:
```
Comment (ros1995) [inline README.md:33]: Nice example of a typical error message so the user knows what to expect

File: README.md [hunk 9]
```diff
@@ -1 +1,91 @@
...
+
+Without a valid API Key, you will receive a 401 Status Code with the following response:
+
+
+
+### Errors
+
+When an error occurs during an API request, you will receive:
+
+An HTTP error status (in the 400-500 range)
```
Comment (ros1995) [inline README.md:30]: Maybe create another section about the successful status codes (e.g. 200 range)

**Pull request merged**